ARSIS-Telemetry-GroundControl/telemetry-api/app/routers/user.py

```diff
@@ -29,14 +31,16 @@ async def get_user(user: str):
     with connection.cursor() as db:
         db.execute("SELECT * FROM users WHERE id = %s;", (user,))
         (id, name, createdAt) = db.fetchone()
-        if (id and name and createdAt):
-            return { "id": id, "name": name, "createdAt": createdAt }
+        if id and name and createdAt:
+            return {"id": id, "name": name, "createdAt": createdAt}
         else:
             return status.HTTP_404_NOT_FOUND
 
+
 @router.put("/")
 async def put_user(user: User):
     with connection.cursor() as db:
         db.execute("INSERT INTO users (name) VALUES (%s) RETURNING *;", (user.name,))
         (id, name, createdAt) = db.fetchone()
-        return { "id": id, "name": name, "createdAt": createdAt}
+        connection.commit()
```

Comment:
What is this connection.commit() for? I haven't seen this on other routes... it looks like sqlalchemy

Reply:
This actually commits the transaction to the DB. I think the current connection will retain the changes in its session but other sessions can't see it. 
Use DBeaver or something similar and add users without the commits and see look at the tables and you wont see the changes.